From 8e6033b77438b58005cd7cf9d37dbc78e496f9e9 Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Sat, 18 Nov 2017 11:58:39 +0100
Subject: [PATCH] Add recipe for plasma-phone-components

commit 18af404679a4cf537adcd328a4dec8d63f2d9b7e

    Fix cross-compilation

    find_file() changes behavior during cross-compilation and only considers
    files from the target sysroot. Instead, use the fact that the file we
    are looking for is always right next to us.

---
 cmake/modules/FindPythonLibrary.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindPythonLibrary.cmake b/cmake/modules/FindPythonLibrary.cmake
index 3c64dab5..c9edd9de 100644
--- a/cmake/modules/FindPythonLibrary.cmake
+++ b/cmake/modules/FindPythonLibrary.cmake
@@ -32,7 +32,7 @@ else()
   FIND_PACKAGE(PythonInterp ${PythonLibrary_FIND_VERSION} QUIET)
 
   if(PYTHONINTERP_FOUND)
-    FIND_FILE(_find_lib_python_py FindLibPython.py PATHS ${CMAKE_MODULE_PATH})
+    SET(_find_lib_python_py ${CMAKE_CURRENT_LIST_DIR}/FindLibPython.py)
 
     EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} ${_find_lib_python_py} OUTPUT_VARIABLE python_config)
     if(python_config)
